diff --git a/ltrace-0.7.3/ltrace-elf.c b/ltrace-0.7.3-mod/ltrace-elf.c
index c571d2a..a3ec79e 100644
--- a/ltrace-0.7.3/ltrace-elf.c
+++ b/ltrace-0.7.3-mod/ltrace-elf.c
@@ -202,7 +202,7 @@ elf_get_section_named(struct ltelf *lte, const char *name,
 				  &name_p, &data);
 }
 
-static int
+/*static int
 need_data(Elf_Data *data, GElf_Xword offset, GElf_Xword size)
 {
 	assert(data != NULL);
@@ -212,15 +212,15 @@ need_data(Elf_Data *data, GElf_Xword offset, GElf_Xword size)
 		return -1;
 	}
 	return 0;
-}
+}*/
 
 #define DEF_READER(NAME, SIZE)						\
 	int								\
 	NAME(Elf_Data *data, GElf_Xword offset, uint##SIZE##_t *retp)	\
 	{								\
-		if (!need_data(data, offset, SIZE / 8) < 0)		\
+		/*if (!need_data(data, offset, SIZE / 8) < 0)		\
 			return -1;					\
-									\
+		*/							\
 		if (data->d_buf == NULL) /* NODATA section */ {		\
 			*retp = 0;					\
 			return 0;					\
